physical_properties/logP/README.md

```diff
@@ -21,4 +21,5 @@ Experimental log *P* measurements was made available in this repository after th
 - `/example_experimental_data/` - This directory contains the experimental report of pH-metric log *P* measurement of phenol with Sirius T3 as an example.
 - `/experimental_data/` - Experimental measurements of log *P* values.
 - `/predictions/` - Prediction files submitted to SAMPL6 log *P* challenge. Reference calculations are also present, but have submission IDs containing the tag `REF` to distinguish them from other submissions
-- `/analysis/` - Analysis of log *P* predictions.
+- `/analysis/` - Analysis of log *P* predictions of challenge molecules.
+- `/analysis_of_extra_molecules/` - Analysis of log *P* predictions of extra molecules (not a part of SAMPL6 log *P* challenge) with reference calculation methods.
```

Comment:
Should this say where these extra molecules came from? (Ignore this if adequately explained in other files.)

Reply:
(But maybe say where it is xplained.)

Reply:
OK. Done.